src/rust/runtime/mod.rs

```diff
@@ -454,6 +391,34 @@ impl SharedDemiRuntime {
         trace!("Check address in use: {:?}", socket_addrv4);
         self.socket_id_to_qdesc_map.is_in_use(socket_addrv4)
     }
+
+    pub fn poll_background_tasks(&mut self) {
+        let background_group_id: TaskId = self.background_group_id;
```

Comment:
This can probably be inlined.

Reply:
ah, this is just needed for ownership. Not sure if there is a more efficient way to write this.